sites/friday/src/projects/region-migration.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Change a project's region
+sidebarTitle: Change regions
+description: See how to change the region your project is in and why you might want to do so.
+---
+
+To host your project data, {{% vendor/name %}} offers several [regions](../development/regions.md).
+You specify a region when you create a project.
+
+You can also change the project's region after it's created.
+
+## Why migrate between regions
+
+- Different data centers are located in different geographic areas.
+  You may want your site close to your users for improved performance.
+- You may want to move to a region with a lower [environmental impact](../development/regions.md#environmental-impact).
+- Some regions are running older versions of the {{% vendor/name %}} orchestration system that offers fewer features.
+  If you are on one of those legacy regions, you can migrate to one of the newer regions.
+
+## 1. Plan the migration
+
+Before starting the migration process, you need to plan for it:
+
+- Plan a time frame in which to handle the migration.
+  Your code shouldn't change during this time to ensure all changes are copied to the new project.
+  Prepare for a brief site outage when you migrate, just as with a relaunch of a site.
+- Set your DNS Time-to-Live as low as possible.
+  This ensures the switch to the new site propagates as quickly as possible.
+
+## 2. Create a new project
+
+In the target region, [create a new project from scratch]({{% create-project-link scratch=true %}}).
+
+If you plan to test for long, start with a Development plan and upsize it before switching the DNS.
+Otherwise, use the desired plan size from the start.
+
+## 3. Add code and environments
+
+{{< codetabs >}}
+
++++
+title=Without a source integration
++++
+
+1. Clone your existing project with Git.
+2. In the new clone, add a remote for the project:
+
+   ```bash
+   {{% vendor/cli %}} project:set-remote
+   ```
+
+   Select your newly created blank project.
+
+3. Push the code for your production branch:
+
+   ```bash
+   {{% vendor/cli %}} push --target {{% variable "PRODUCTION_BRANCH_NAME" %}}
+   ```
+
+4. (Optional) Checkout other branches and then push their code:
+
+   ```bash
+   {{% vendor/cli %}} push --activate --target {{% variable "BRANCH_NAME" %}} --parent {{% variable "PRODUCTION_BRANCH_NAME" %}}
+   ```
+
+<--->
+
++++
+title=With a source integration
++++
+
+For a [source integration](../integrations/source/_index.md) with GitHub, BitBucket, or GitLab,
+add the integration to your new project.
+Your new project then mirrors the configured repository automatically.
+
+{{< /codetabs >}}
+
+## 4. Copy files
+
+If you have files in a mount, first download them:
+
+```bash
+{{% vendor/cli %}} mount:download
+```
+
+Then upload them to your new project:
+
+```bash
+{{% vendor/cli %}} mount:upload
```

Comment:
`mount:upload` alone is not working so maybe display the doc page snippet?
```
{{% vendor/cli %}} mount:upload --mount private --source ./private
```

Reply:
@flovntp I'm not following. `mount:upload` and `mount:download` will interactively prompt you for which mount, as well as the target directory (`source`).